packs/feat-effects/effect-curse-of-the-heros-burden.json

```diff
@@ -10,6 +10,7 @@
                 "Major",
                 "Extreme"
             ],
+            "loop": false,
```

Comment:
Do we need to strip this on extract?

Reply:
Looks like we do. Let me remove the cruft from this one. It got hit by me opening items to test some user complaints then extracting all since I was hitting a lot of compendiums.